### DISCUSSION TITLE
Electrode Compositions for Lithium-Ion Chemistries

### DISCUSSION THREAD

**Discussion by Pratik260199** [pybamm-team/PyBaMM#1558] [Q&A] [answered]
What are the exact compositions of negative and positive electrodes used in all the parameter sets in PyBaMM ? For example, Chen2020 uses NMC811. Similarly, where can we find the exact compositions for all the other parameter sets?

**Comment by brosaplanella** [comment, marked as answer]
I would say the best way to check the exact compositions is to check the papers where the parameters come from. You can find the relevant references in the README of each parameter set. For example, [here is the README for the Chen2020 positive electrode](https://github.com/pybamm-team/PyBaMM/tree/develop/pybamm/input/parameters/lithium_ion/positive_electrodes/nmc_Chen2020) (see bottom of the page).
